index.js

```diff
@@ -2,16 +2,35 @@
 const path = require('path');
 const eslint = require('eslint');
 const globby = require('globby');
+const multimatch = require('multimatch');
 const optionsManager = require('./options-manager');
 
 exports.lintText = (str, opts) => {
 	opts = optionsManager.preprocess(opts);
 
+	if (opts.cwd && opts.filename) {
```

Comment:
Done! Guess this is a breaking change as passing `ignores` before these changes did nothing.